ocs_samples/basic_samples/Authentication/ImplicitFlow/DotNet/ImplicitFlow/wwwroot/app.js

```diff
@@ -16,7 +16,7 @@ function log() {
 
 document.getElementById("login").addEventListener("click", login, false);
 document.getElementById("logout").addEventListener("click", logout, false);
-document.getElementById("tenant").addEventListener("click", getTenant, false);
+document.getElementById("users").addEventListener("click", getUsers, false);
```

Comment:
Test should be updated to reference the ID tenant instead of users

Reply:
@derekendres Get Tenant endpoint is not part of the public api documentation. As such, I updated all the tests to make a call for the Get Users endpoint instead.